content/blog/how-to-use-custom-partition-tables-on-esp32/index.md

```diff
@@ -0,0 +1,309 @@
+---
+title: "How to use custom partition tables on ESP32"
+date: 2021-02-13
+showAuthor: false
+authors:
+    - "pedro-minatel"
+---
+# How to use custom partition tables on ESP32
+
+
+# Introduction
```

Comment:
Apparently, this heading and all the other ones below should be deeper by one level respectively.

```suggestion
## Introduction
```